massbalancemachine/data_processing/create_glacier_grid.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+import salem
+import pyproj
+import pandas as pd
+
+def create_glacier_grid(ds, years, glacier_indices, gdir, rgi_gl):
```

Comment:
Can you please add there the parameter types and return type?

Reply:
This is now added in new commit.